src/components/metadata/index.js

```diff
@@ -7,22 +7,39 @@ import CopyIcon from '../icons/copy';
 import CloseIcon from '../icons/close';
 import MetaDataRow from './metadata-row';
 import MetaDataValue from './metadata-value';
+import MetaDataCode from './metadata-code';
+import MetaCodeToggle from './metadata-code-toggle';
 import {
   getVisibleMetaSidebar,
   getClickedNodeMetaData
 } from '../../selectors/metadata';
 import { toggleNodeClicked } from '../../actions/nodes';
+import { toggleCode } from '../../actions';
 import './styles/metadata.css';
 
 /**
  * Shows node meta data
  */
-const MetaData = ({ visible = true, metadata, onToggleNodeSelected }) => {
+const MetaData = ({
+  visible = true,
```

Comment:
I wonder what is this `visible` value for?

Reply:
Yeah as @richardwestenra says, see also `mapStateToProps` in that function as that's how it gets controlled.